pynder/models/user.py

```diff
@@ -10,18 +10,30 @@ def __init__(self, data, session):
         self._data = data
         self.id = data['_id']
 
-        SIMPLE_FIELDS = "name bio birth_date common_friends common_likes ping_time".split(" ")
+        SIMPLE_FIELDS = "name bio gender birth_date common_interests common_connections ping_time".split(" ")
```

Comment:
gender, common_interests and common_connections shouldn't be in SIMPLE_FIELDS since they need additional processing below. Other than this the PR looks good, if you fix this I'll go ahead and merge it.